pkgs/by-name/fo/follow/package.nix

```diff
@@ -30,19 +30,18 @@ stdenv.mkDerivation rec {
 
   pnpmDeps = pnpm_9.fetchDeps {
     inherit pname version src;
-    hash = "sha256-FzMjN0rIjYxexf6tix4qi3mnuPkadjKihhN0Pj5y2nU=";
+    hash = "sha256-xNGLYzEz1G5sZSqmji+ItJ9D1vvZcwkkygnDeuypcIM=";
   };
 
   env = {
     ELECTRON_SKIP_BINARY_DOWNLOAD = "1";
 
     # This environment variables inject the production Vite config at build time.
     # Copy from:
-    # 1. https://github.com/RSSNext/Follow/blob/v0.2.0-beta.2/.github/workflows/build.yml#L18
+    # 1. https://github.com/RSSNext/Follow/blob/v0.3.7/.github/workflows/build.yml#L18
```

Comment:
it seem the version find-replace clobbers this url, please update it to reflect the original location

Reply:
> it seem the version find-replace clobbers this url, please update it to reflect the original location

It's the original location, the gh action might change between versions.

Reply:
I went through the git log, this is the initial url:

https://github.com/RSSNext/Follow/blob/0745ac07dd2a4a34e4251c034678ace15c302697/.github/workflows/build.yml#L18

vs proposed

https://github.com/RSSNext/Follow/blob/v0.3.7/.github/workflows/build.yml#L18

of note is which line is highlighted

Reply:
> I went through the git log, this is the initial url:
> 
> [RSSNext/Follow@`0745ac0`/.github/workflows/build.yml#L18](https://github.com/RSSNext/Follow/blob/0745ac07dd2a4a34e4251c034678ace15c302697/.github/workflows/build.yml#L18)
> 
> vs proposed
> 
> [RSSNext/Follow@`v0.3.7`/.github/workflows/build.yml#L18](https://github.com/RSSNext/Follow/blob/v0.3.7/.github/workflows/build.yml?rgh-link-date=2025-03-11T23%3A44%3A05Z#L18)
> 
> of note is which line is highlighted

Oh, I get it. I miss to update the line number.